commit 47684b6b76f1e3248557506725d7ca0fae613ec6
Author: Trent Mick <trentm@gmail.com>
Date:   2019-05-27T12:02:39-07:00 (4 months ago)
    
    TOOLS-2262 eng.git check-eslint should not re-install eslint every time

diff --git a/tools/mk/Makefile.targ b/tools/mk/Makefile.targ
index a9e1044..7be1d5c 100644
--- a/tools/mk/Makefile.targ
+++ b/tools/mk/Makefile.targ
@@ -242,7 +242,7 @@ check-json: $(JSON_FILES:%=%.jsonchk)
 
 ESLINT_TARGET = $(if $(ESLINT_FILES), check-eslint)
 .PHONY: check-eslint
-check-eslint: $(ESLINT_EXEC)
+check-eslint: | $(ESLINT_EXEC)
 	$(ESLINT) $(ESLINT_FLAGS) $(ESLINT_FILES)
 
 JSL_NODE_TARGET = $(if $(JSL_FILES_NODE), check-jsl-node)
